plugin/trino-kafka/pom.xml

```diff
@@ -339,6 +339,45 @@
                     </ignoredResourcePatterns>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
nit: Can we add a issue to remove this workaround on generating the code ? The way we invoke parser should support it out of the box, I have a rough fix for it and will be working on a PR for the same.

Reply:
Sure, I'll create an issue + add TODO.

cc: @adamjshook regarding some schema parser improvements Praveen plans to make which might be relevant for you.

Reply:
Yup, that'd be relevant as I am using the same plugin. I'll keep an eye out for it.